inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -230,15 +230,49 @@ def simulate(
         resource_pool_id: typing.Optional[uuid.UUID] = None,
         run_async: bool = False,
         plasma_surface_filepath: typing.Optional[types.Path] = None,
+        num_iterations: int = None,
+        num_samples: int = None,
+        sigma_scaling_factor: float = None,
     ) -> tasks.Task:
         """Simulates the scenario.
 
+        A simulation can be performed in two ways simply by adding some more
+        arguments. 
+        
+        If `num_iterations`, `num_samples` and `sigma_scaling_factor`
+        are all not provided, the simulation merely uses the created device and 
+        performs the computation of the magnetic field produced by its coils
+        on the given plasma surface. Additionally, it produces an output of
+        the objective functions that assess the quality of the device.
+
+        Alternatively, if the arguments are provided, the simulation process
+        is different. It also uses the create device, but as a beginning step.
```

Comment:
We need to explain with 1/2 more sentences what this "search" process is about. Shall we use the word "optimization"?